tests/table-operations.spec.ts

```diff
@@ -490,3 +509,165 @@ test('rollup rows and aggregrate columns', async ({ page }) => {
+  const table2Name = generateVarName('t2');
+
+  await test.step('create 2nd table', async () => {
+    const consoleInput = page.locator('.console-input');
+    await consoleInput.click();
+
+    const command = makeTableCommand(table2Name, TableTypes.AllTypes);
+
+    await pasteInMonaco(consoleInput, command);
+    await page.keyboard.press('Enter');
+  });
+
+  await test.step('drag table to new panel', async () => {
+    // opening up this menu makes it easier to drag to that corner
+    await page
+      .getByTestId('btn-iris-grid-settings-button-table')
+      .first()
+      .click();
+
+    const table = page
+      .locator('.lm_tab')
+      .filter({ has: page.getByText(table2Name) });
+    const target = page.getByText('Command History');
+    const dropIndicator = page.locator('.lm_dragProxy');
+    await dragComponent(table, target, dropIndicator, 300);
+
+    await page.getByTestId('btn-page-close').first().click();
+    await page.getByTestId('btn-page-close').nth(1).click();
+
+    await waitForLoadingDone(page);
+  });
+
+  await test.step('add input filter to int column', async () => {
+    await page.getByRole('button', { name: 'Controls' }).click();
+
+    const inputFilter = page.getByRole('button', { name: 'Input Filter' });
+    const target = page.getByText('Command History');
+    const dropIndicator = page.locator('.lm_dragProxy');
+    await dragComponent(inputFilter, target, dropIndicator);
+  });
+
+  await test.step('add linker filter to string column', async () => {
+    await page.getByRole('button', { name: 'Controls' }).click();
+    await page.getByRole('button', { name: 'Linker' }).click();
+
+    // Note: do not have to drag to use linker filter I just wanted it in this position
+    const firstStringCol = page.locator('.iris-grid .grid-wrapper').first();
+    await firstStringCol.click({ position: { x: 20, y: 10 } });
+    const secondStringCol = page.locator('.iris-grid .grid-wrapper').nth(1);
+    await secondStringCol.click({
+      position: { x: 20, y: 10 },
+    });
+
+    await page
+      .locator('.linker-toast-dialog')
+      .getByRole('button', { name: 'Done' })
+      .click();
+  });
+
+  await test.step('use linker filter', async () => {
+    await page
+      .locator('.iris-grid .grid-wrapper')
+      .first()
+      .dblclick({
+        position: { x: 20, y: 60 },
+      });
+
+    await waitForLoadingDone(page, 1);
+  });
+
+  await test.step('use input filter', async () => {
+    await page.getByRole('combobox').selectOption({ label: 'Int' });
+    await page.getByRole('button', { name: 'Save' }).click();
+
+    await expect(page.getByPlaceholder('Enter value...')).toHaveCount(1);
+    await page.getByPlaceholder('Enter value...').click();
+    await page.keyboard.type('>1000');
+    await expect(page.getByPlaceholder('Enter value...')).toHaveValue('>1000');
+
+    await waitForLoadingDone(page);
+    await waitForLoadingDone(page, 1);
+    await artificialWait(page);
```

Comment:
Really the wait should be before the waitForLoadingDone, since that will fire immediately.